mlflow/utils/file_utils.py

```diff
@@ -413,3 +414,13 @@ def yield_file_in_chunks(file, chunk_size=100000000):
                 yield chunk
             else:
                 break
+
+
+def download_file_using_signed_uri(signed_uri, download_path, chuck_size=100000000):
```

Comment:
Yes, it would work for any HTTP uri. 
Not sure about explicitly supporting HTTP, but we can add it for now. Along with a comment saying it is being used to download presigned urls.